README.md

```diff
@@ -44,14 +44,24 @@ Now when you clone and start working on a new project, you can  run `sb` to inst
 
 - [nvm](https://github.com/creationix/nvm) - Installs the Node version defined in `.nvmrc` during `bootstrap`
 
+- [pyenv](https://github.com/pyenv/pyenv) - Installs the Python version defined in `.python-verson` during `bootstrap`
+
 - [rbenv](https://github.com/rbenv/rbenv) - Installs the Ruby version defined in `.ruby-verson` during `bootstrap`
 
 - [bundler](http://bundler.io/) - Installs Ruby dependencies declared in `Gemfile` during `bootstrap`
 
+- [django](http://bundler.io/) - Uses `manager.py` to run the server, tests, and console
```

Comment:
This should probably link to https://www.djangoproject.com/?

Reply:
This PR looks awesome. I think this should also be `manage.py` on this line too.